### ISSUE TITLE
The latest version of VS Code (1.78) is incompatible with the Azure Functions extension (v1.10.5)

### ISSUE THREAD

**Issue by CorbanAI** [microsoft/vscode#181608]
Does this issue occur when all extensions are disabled?: The issue is with an extension's compatibility.

- VS Code Version: 1.78
- OS Version: Windows_NT x64 10.0.19045

Steps to Reproduce:

1. Have VS Code 1.78 and Azure Functions extension v1.10.5
2. Try to debug a function-app written in C#
3. When starting the development host, the package windows-process-tree cannot be located.

Rolled back to the previous version of VS Code and debugging worked as expected.

**Comment by VSCodeTriageBot** [comment]
Thanks for creating this issue! It looks like you may be using an old version of VS Code, the latest stable release is 1.78.0. Please try upgrading to the latest version and checking whether this issue remains.

Happy Coding!

**Comment by CorbanAI** [comment]
I had the latest version of VS Code (as mentioned in the issue reporting) and that surfaced the problem.
Rolled back to previous version to bypass the problem until resolved.


Corban Neeley
Senior Software Developer


[cid:logo-master-mono-black_38e1a4bb-0d97-4f33-8f7d-4237159c3186.png]   1300 513 888
D: 0432 872 414
Level 4, 1 Breakfast Creek Road, Newstead QLD 4006
***@***.***
www.urbanx.io<http://www.urbanx.io>



<https://urbanx.io/christmas-gift-appeal-2022/>

**Comment by gjsjohnmurray** [comment]
/causedByExtension

https://github.com/microsoft/vscode-azurefunctions/issues/3664

**Comment by VSCodeTriageBot** [comment]
This issue is caused by an extension, please file it with the repository (or contact) the extension has linked in its overview in VS Code or the [marketplace](https://aka.ms/vscodemarketplace) for VS Code. See also our [issue reporting guidelines](https://aka.ms/vscodeissuereporting). If you don't know which extension is causing the problem, you can run `Help: Start extension bisect` from the command palette (F1) to help identify the problem extension.

Happy Coding!

**Comment by zadirion** [comment]
Don't mean to be rude, but this kind of change that breaks extensions isn't something that belongs in a minor version update. One that happened without us even noticing due to the automatic update.
And if it did, it's definitely not something that needs to be fixed by the extension, you'd never expect a minor version increment to break any kind of compatibility.
It would be reasonable to expect the extension to need to implement an update if you released a major version (x+1.y.z) or even middle, ok sure, (x.y+1.z), but an (x.y.z+1)? This should be something fixed vscode side imo
I did just waste an entire day thinking I broke somehing, never suspected a stealth vscode update breaking this.
